tools/cldr-apps/src/main/webapp/WEB-INF/tmpl/manage.jsp

```diff
@@ -55,9 +52,11 @@ String helpName = subCtx.getString("helpName");
                 subCtx.println("<br> (Note: in the Vetting phase, you may not submit new data.) ");
             } else if(SurveyMain.isPhaseClosed() && !UserRegistry.userIsTC(subCtx.session.user)) {
                 subCtx.println("<br>(SurveyTool is closed to vetting and data submissions.)");
+            } else if (SurveyMain.isPhaseFinalTesting() && !UserRegistry.userIsTC(subCtx.session.user)) {
+                subCtx.println("(SurveyTool is in the Final Testing phase; you may not vote.)");
             }
```

Comment:
This is the most essential addition. The other changes in this file fix compiler warnings or delete dead ("if false") code